priv/repo/migrations/20171220030918_create_users.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Hades.Repo.Migrations.CreateUsers do
+  use Ecto.Migration
+
+  def change do
+  	create table(:users) do
+  		add :email, :string
+  		add :name, :string
+  		add :password_hash, :string
+  		add :auth_token, :string
+  		add :is_admin, :boolean
+
+  		timestamps()
+  	end
+
+  	create unique_index(:users, [:email])
+  end
+end
```

Comment:
I don't know why the indentation got so weird but if we copy and paste the code here http://elixirplayground.com/ it looks like there's no problem.